hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeHttpServer.java

```diff
@@ -93,6 +99,27 @@ public static void initWebHdfs(Configuration conf, HttpServer2 httpServer2,
 
     final String pathSpec = WebHdfsFileSystem.PATH_PREFIX + "/*";
 
+    if (conf.get(HADOOP_SECURITY_AUTHENTICATION,
+        "simple").equalsIgnoreCase("kerberos") &&
+        conf.get(HADOOP_HTTP_AUTHENTICATION_TYPE,
+            "simple").equalsIgnoreCase("simple") &&
```

Comment:
Why this is simple?

Reply:
@zeekling Thanks for your review. In HADOOP-16314, the HTTP AuthenticationFilter was unified, and we would also like to follow this configuration here.